connectors/src/connectors/zendesk/temporal/activities.ts

```diff
@@ -293,27 +310,102 @@ export async function syncZendeskCategoryActivity({
 /**
  * This activity is responsible for syncing all the articles in a Category.
  * It does not sync the Category, only the Articles.
- *
- * @returns true if the Category was updated, false if it was deleted.
  */
-// eslint-disable-next-line no-empty-pattern
-export async function syncZendeskArticlesActivity({}: {
+export async function syncZendeskArticlesActivity({
+  connectorId,
+  categoryId,
+  currentSyncDateMs,
+  forceResync,
+}: {
   connectorId: ModelId;
   categoryId: number;
   currentSyncDateMs: number;
   forceResync: boolean;
-}) {}
+}) {
+  const connector = await _getZendeskConnectorOrRaise(connectorId);
+  const dataSourceConfig = dataSourceConfigFromConnector(connector);
+  const loggerArgs = {
+    workspaceId: dataSourceConfig.workspaceId,
+    connectorId,
+    provider: "zendesk",
+    dataSourceId: dataSourceConfig.dataSourceId,
+  };
+  const configuration = await _getZendeskConfigurationOrRaise(connectorId);
+  const categoryInDb = await _getZendeskCategoryOrRaise({
+    connectorId,
+    categoryId,
+  });
```

Comment:
nit: looks like the 4 awaited calls could be done in parallel (Promise.all([...]))